cloudoperators/juno
# chore(ui): integrate vite for ui-components

# Summary

This pull request transitions the project from Rollup to Vite as the primary bundler to improve build performance and developer experience. It maintains backward compatibility while ensuring existing workflows remain functional. The Storybook configuration remains untouched for now, with plans to update it in a subsequent iteration.

# Changes Made

- Removed Rollup as the bundler.
- Fully integrated Vite as the new bundler across the project.
- Ensured backward compatibility with existing configurations and workflows.

# Related Issues

- Issue: https://github.com/cloudoperators/juno/issues/444

# Screenshots (if applicable)



# Testing Instructions

1. Run `npm install` to install updated dependencies.
2. Execute `npm run build` to verify the Vite setup.
3. Test backward compatibility with previous configurations.

# Checklist

- [ ] Performed a self-review of the changes.
- [ ] Verified backward compatibility with existing setups.
- [ ] Added comments for any complex sections of code.
- [ ] Updated or ensured no changes were required for documentation.
- [ ] Tested locally, and all unit tests pass.
- [ ] Ensured no new warnings or errors are introduced.

This marks the initial step toward optimizing our build process with Vite while keeping Storybook configurations pending for a future update.
